authors/apps/profiles/test_follow.py

```diff
@@ -0,0 +1,108 @@
+from django.contrib.auth import get_user_model
+from rest_framework.test import APITestCase
+from rest_framework.reverse import reverse
+from rest_framework import status
+from .models import Profile
+
+
+from authors.apps.authentication.models import UserManager
+
+
+class ProfileTest(APITestCase):
+    """
+    User authentication test cases
+    """
+    def setUp(self):
+        """
+        Method for setting up user
+        """
+        self.user =  { 
```

Comment:
Let's have this styles as: 
```python
        self.user =  { 
            "user":{
                'username': 'janeDoe', 
                'email': 'jane@doe.com', 
                'password': 'janedoe123', 
            }
        }

Reply:
I'm thinking what Edward is saying for the indentation use 4 spaces instead of 8 spaces, isn't that right @codingedward ?

Reply:
@codingedward @FawazFarid, just had the indentation changed to four spaces.

Reply:
Yes, that's right @FawazFarid.

Cool @michael-basweti.